javascript-new/example-constructor.js

```diff
@@ -0,0 +1,15 @@
+// function using the new keyword ExampleConstructor
+
+function ExampleConstructor(one) {}
+
+console.log(ExampleConstructor.prototype);
+console.log(`${typeof ExampleConstructor.prototype}`);
+
+// calling the func with a new as an arg
+
+const newKeywordArg = new ExampleConstructor();
+
+console.log(
+  'The typeof in the function with a new argument is a:',
+  newKeywordArg
+);
```

Comment:
This is incorrect. You should be using the `instanceof` operator to check if `newKeywordArg` is an instance of `ExampleConstructor`